instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/SpanSuppressors.java

```diff
@@ -85,4 +87,28 @@ public boolean shouldSuppress(Context parentContext, SpanKind spanKind) {
       return true;
     }
   }
+
+  static class ByContextKey implements SpanSuppressor {
+    private final SpanSuppressor delegate;
+    static final ContextKey<Boolean> SUPPRESS_INSTRUMENTATION =
+        ContextKey.named(
+            "suppress_instrumentation"); // todo must use the key defined in the Java SDK.
```

Comment:
I was planning to use the tool recently added to the core project, however, I just realized that my changes are part of the `opentelemetry-exporter-common` lib which this project currently doesn't have as part of its dependencies. So I'm wondering if it's ok to add said dependency here or if it's better to move the context suppressing utility over to the `opentelemetry-api` core subproject?

Reply:
oh no, that's a good point

we could access it via reflection for now, but this seems like a good reason to move the utility over to `opentelemetry-api` in the long run

Reply:
Got it 👍 I've updated it to use reflection for now.